doc/man3/OPENSSL_gmtime.pod

```diff
@@ -25,11 +25,14 @@ OPENSSL_tm_to_posix - platform-agnostic OpenSSL time routines
 
 =head1 DESCRIPTION
 
-OPENSSL_gmtime() returns the UTC time specified by I<timer> into the
-provided I<result> argument. |timer| must be in the range of year 0 to
-9999.  Only the fields I<tm_year>, I<tm_mon>, I<tm_mday>, I<tm_hour>,
-I<tm_min>, and I<tm_sec> will be updated  in I<result> on success,
-all other fields will be zeroed.
+OPENSSL_gmtime() returns the Unix/Posix time specified by I<timer>
+into the provided I<result> argument. |timer| must be a value of the
+number of seconds relative to 0 hours, 0 minutes, 0 seconds, January 1,
+1970, Coordinated Universal Time, without including leap seconds, and
+must be in the range of the year 0 to 9999 inclusively. I<result> will
```

Comment:
I'd suggest:
```suggestion
must be in the range of the year 1970 to 9999 inclusively. I<result> will
```

This avoids problematic cases and covers everything that should be cryptographically relevant.